### DISCUSSION TITLE
Not scrolling to top

### DISCUSSION THREAD

**Discussion by azamat-747** [inertiajs/inertia#1665] [Help (Vue)]
### Version:

- `@inertiajs/vue3` version: 1.0.0

### Describe the problem:

When i click link in footer, inertia not scrolling to top

`<footer> <Link :href="route('home')">Home</Link> </footer>`

[Video](https://drive.google.com/file/d/1cYoUA1abnJBRsMjDFjm3BPTC07sTTOIY/view?usp=sharing)

**Comment by reinink** [comment]
Hey, really hard to help here without a minimal reproduction, plus the linked-to video doesn't work (404). I just converted this to a discussion, as I suspect this isn't a bug and likely just an implementation issue, but happy to have another look if you can share a reproduction 👍

**Reply by azamat-747** [reply]
Sorry for the link, updated you can check

**Reply by reinink** [reply]
Hey thanks for sharing — again, without a *minimal* reproduction as a Git repo, it's almost impossible for us to help here.

The only thing that comes to mind is that you might be use a scroll area instead of full page scroll. If that's the case, check out our scroll regions documentation:

https://inertiajs.com/scroll-management#scroll-regions
